packages/gatsby-theme-docs/src/layouts/content.js

```diff
@@ -61,11 +61,13 @@ const LayoutContent = (props) => {
             allowWideContentLayout={props.pageData.allowWideContentLayout}
           >
             <LayoutGlobalNotification>
-              {props.pageData.globalNotification && (
+              {siteData.siteMetadata.globalNotification.content && (
```

Comment:
related to the above discussion concerning defaults:  the code is assuming "globalNotification" is always set but is just checking for "truthyness".